turbo/jsonrpc/send_transaction.go

```diff
@@ -68,20 +68,16 @@ func (api *APIImpl) SendRawTransaction(ctx context.Context, encodedTx hexutility
 	api.SenderLocks.AddLock(sender)
 	defer api.SenderLocks.ReleaseLock(sender)
 
-	if txn.Type() != types.LegacyTxType {
-		latestBlock, err := api.blockByNumber(ctx, rpc.LatestBlockNumber, tx)
-
-		if err != nil {
-			return common.Hash{}, err
-		}
-
-		if !cc.IsLondon(latestBlock.NumberU64()) {
+	// When we are not in normalcy, we need to check if the transaction is a legacy transaction
+	if !cc.IsNormalcy(header.NumberU64()) && txn.Type() != types.LegacyTxType {
+		if !cc.IsLondon(header.NumberU64()) {
```

Comment:
do we need the london check here?  if we're not running in normalcy then we only support legacy and can just return an error at this point